Packs/Hackuity/Integrations/Hackuity/Hackuity.yml

```diff
@@ -0,0 +1,228 @@
+    - *argument_trs_max
+    - *argument_vuln_type
+    deprecated: false
+    description: Search for vulndb vulnerabilities in Hackuity.
+    name: hackuity-search-vulndb-vulnerabilities
+    outputs: &outputs_vulnerabilities
+    - contextPath: Hackuity.Vulnerabilities.Attributes
+      description: The attributes of the vulnerability
+      type: string
+    - contextPath: Hackuity.Vulnerabilities.Score.CVSS
+      description: The CVSS of the vulnerability
+      type: string
+    - contextPath: Hackuity.Vulnerabilities.Score.TRS
+      description: The TRS of the vulnerability
+      type: string
+    - contextPath: Hackuity.Vulnerabilities.Description
+      description: The description of the vulnerability
+      type: string
+    - contextPath: Hackuity.Vulnerabilities.ID
+      description: The ID of the vulnerability
+      type: string
+    - contextPath: Hackuity.Vulnerabilities.Seen.First
+      description: The date of the first time the vulnerability has been seen
+      type: date
+    - contextPath: Hackuity.Vulnerabilities.Findings.Total
+      description: The total number of findings on this vulnerability
+      type: string
+    - contextPath: Hackuity.Vulnerabilities.Findings.Open
+      description: The number of open findings on this vulnerability
+      type: string
+    - contextPath: Hackuity.Vulnerabilities.Findings.Closed
+      description: The number of closed findings on this vulnerability
+      type: string
+    - contextPath: Hackuity.Vulnerabilities.Findings.Ignored
+      description: The number of ignored findings on this vulnerability
+      type: string
+    - contextPath: Hackuity.Vulnerabilities.VulnType.Id
+      description: The ID of the vulnerability type
+      type: string
+    - contextPath: Hackuity.Vulnerabilities.VulnType.Name
+      description: The name of the vulnerability type
+      type: string
+  - arguments:
+    - *argument_asset_name
+    - *argument_asset_type
+    - *argument_attribute
+    - *argument_cvss_min
+    - *argument_cvss_max
+    - *argument_limit
+    - *argument_trs_min
+    - *argument_trs_max
+    - *argument_vuln_type
+    deprecated: false
+    description: Search for provider vulnerabilities in Hackuity.
+    name: hackuity-search-provider-vulnerabilities
+    outputs: *outputs_vulnerabilities
+  - deprecated: false
+    description: List the widgets in the default dashboard of the user.
+    name: hackuity-dashboard-widgets
+    #outputs: not documented: may change in the future
+  - arguments:
+    - default: false
+      description: The ID of the widget
+      isArray: false
+      name: widget_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Get the data of a dashboard widget
+    name: hackuity-dashboard-data
+    #outputs: not documented: may change in the future
+  isfetch: false
+  runonce: false
+  script: '-'
+  type: python
+  subtype: python3
+  dockerimage: demisto/python3:3.10.1.25933
+fromversion: 5.5.0
```

Comment:
since its a new integration its preferred to be supported from 6.0.0, is there a specific reason its set to be 5.5.0?

Reply:
Done